external-crates/move/crates/move-compiler/src/parser/ast.rs

```diff
@@ -104,12 +104,24 @@ pub enum Use {
         ty: Box<NameAccessChain>,
         method: Name,
     },
+    // used for one of the three cases when `LeadingNameAccess` represents `some_pkg`
+    // - `some_pkg`
+    // - `some_pkg::`
+    // - `some_pkg::{`
+    // where first location represents `::` and the second one represents `{`
+    Partial(LeadingNameAccess, Option<Loc>, Option<Loc>),
 }
 
 #[derive(Debug, PartialEq, Clone, Eq)]
 pub enum ModuleUse {
     Module(Option<ModuleName>),
     Members(Vec<(Name, Option<Name>)>),
+    // used for one of the three cases when defining `ModuleUse` for `some_mod`:
+    // - `... some_mod`
+    // - `... some_mod::`
+    // - `... some_mod::{`
+    // where first location represents `::` and the second one represents `{`
+    Partial(Option<Loc>, Option<Loc>),
```

Comment:
Also name the variant fields:
```suggestion
    Partial(colon: Option<Loc>, open_brace: Option<Loc> },
```

Reply:
Done!